### PR TITLE
- Adding affinity for kubernetes-dashboard

### PR DESCRIPTION
**Pull request by ahma**
- version bump

Signed-off-by: Ferenc HERNADI <ferenc.hernadi@gmail.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/kubernetes-dashboard/values.yaml [hunk 1]
```diff
@@ -34,6 +34,11 @@ tolerations: []
 #    value: "value"
 #    effect: "NoSchedule|PreferNoSchedule|NoExecute"
 
+## Affinity
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+affinity: {}
```
Comment (davidkarlsen) [inline stable/kubernetes-dashboard/values.yaml:39]:
```suggestion
affinity: []
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Review by rimusz — COMMENTED** [review]
this commits is missing signoff

**Comment by ahma** [comment]
@davidkarlsen ff I can do anything else please let me  know. 
Thanks Feri!

**Comment by davidkarlsen** [comment]
@ahma There seems to be some unrelated keel changes?

File: stable/kubernetes-dashboard/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: kubernetes-dashboard
-version: 0.7.5
+version: 0.7.6
```
Comment (davidkarlsen) [inline stable/kubernetes-dashboard/Chart.yaml]:
```suggestion
version: 0.8.0
```

new feature

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**